scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenClasses.scala

```diff
@@ -10,18 +10,18 @@ object Keyword {
   def unapply(token: Token): Boolean = {
     token.is[KwAbstract] || token.is[KwCase] || token.is[KwCatch] ||
     token.is[KwClass] || token.is[KwDef] || token.is[KwDo] ||
-    token.is[KwElse] || token.is[KwExtends] || token.is[KwFalse] ||
-    token.is[KwFinal] || token.is[KwFinally] || token.is[KwFor] ||
-    token.is[KwForsome] || token.is[KwIf] || token.is[KwImplicit] ||
-    token.is[KwImport] || token.is[KwLazy] || token.is[KwMatch] ||
-    token.is[KwMacro] || token.is[KwNew] || token.is[KwNull] ||
-    token.is[KwObject] || token.is[KwOverride] || token.is[KwPackage] ||
-    token.is[KwPrivate] || token.is[KwProtected] || token.is[KwReturn] ||
-    token.is[KwSealed] || token.is[KwSuper] || token.is[KwThis] ||
-    token.is[KwThrow] || token.is[KwTrait] || token.is[KwTrue] ||
-    token.is[KwTry] || token.is[KwType] || token.is[KwVal] ||
-    token.is[KwVar] || token.is[KwWhile] || token.is[KwWith] ||
-    token.is[KwYield]
+    token.is[KwElse] || token.is[KwEnum] || token.is[KwExtends] ||
```

Comment:
if you are adding enum only, would it make sense to append it so that only one line changes?

Reply:
Of course, in similar cases in the future, I will simply prefer the number of lines changed over the alphabetical order if it will make sense.

Reply:
didn't realize they are sorted. then keep as is.

Reply:
can be added as the fourth in the line as well

Reply:
However, it is formatted in a strange way then so I will then keep it as it is